src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  padding: 20px;
+  $root: &;
+
+  &__content {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1px;
+    width: $cell-size * 7 + 7;
+    margin: 0 auto;
+  }
+
+  &__day {
+    box-sizing: border-box;
```

Comment:
it s better to apply this property to all elements (use * selector)
when all elements have the same `box-sizing` value, their behavior becomes more predictable